### DISCUSSION TITLE
policy.xml - can it be set to a custom location?

### DISCUSSION THREAD

**Discussion by jetpack1116** [ImageMagick/ImageMagick#5755] [Help] [answered]
ImageMagick seems to look at the policy.xml in its working directory.

Say, given the directory C:\Program Files\ImageMagick-7.1.0-Q16-HDRI\

inside this directory,  there is magick.exe and policy.xml
and magick.exe is looking at policy.xml in this directory

Is there a way to move the policy.xml to a different location and magick.exe can still look into it?
Can it be set in a config file ?

**Comment by urban-warrior** [comment, marked as answer]
Under Windows, ImageMagick searches for configuration files such as `policy.xml` in the following order, and loads them if found:

```
$MAGICK_CONFIGURE_PATH
<windows registry>
$PREFIX/config
$USERPROFILE/.config/ImageMagick
<client path>
```

Above, $PREFIX is the default install path, typically c:\\Program Files\\ImageMagick-7.1.0.

For an uninstalled Windows installation, the configuration load order is:

```
$MAGICK_CONFIGURE_PATH
$MAGICK_HOME
$USERPROFILE/.config/ImageMagick
client path
<current directory>
```

If a configuration file cannot not be found, ImageMagick relies on built-in default values. See https://imagemagick.org/script/resources.php#configure for more information.

**Reply by jetpack1116** [reply]
@urban-warrior 

Thank you for this.

I checked and setting env variable MAGICK_CONFIGURE_PATH works.
